### ISSUE TITLE
Main iframe title does not help navigation

### ISSUE THREAD

**Issue by Sereza7** [cryptpad/cryptpad#1606]
### Description

Iframes should have text alternatives. This is a continuation to https://github.com/cryptpad/cryptpad/issues/1123. 
The main iframe's title is `iframe`, which does not help much with navigation. This solution might fulfill WCAG requirements, it does not improve the accessibility of this structure.

![Screenshot from 2024-08-26 17-31-57](https://github.com/user-attachments/assets/0283cdc0-9f88-42e8-a38d-ef5177017f20)

### Steps to reproduce

1. Create a document
2. Open the document
3. Check the programmatic name of the main iframe, either with a screen reader or with your browser inspector.

### Expected behavior

The main iframe has a proper name, such as `Main content` or `Main`.

### Which operating system are you using?

Linux/BSD/UNIX

### What browsers are you seeing the problem on?

Mozilla Firefox

### Extensions

Too many :')
Ublock and Consent-O-Matic are usually messing some features up but I doubt they'd be relevant here.

### Version

2024.6.1
